README.md

```diff
@@ -60,27 +62,44 @@ DB_PORT='my_db_port'
    └── create_raw_intermediate_indexes.py  
     
 ├── **insert_user_registration/**  
-   └── README.md  
+   └── README.md
+   └── __init__.py  
    └── insert_new_data.py  
     
 ├── **monitoring_data_pipeline/**  
-   └── README.md  
-   └── post_cohort_repeatative_script  
+   └── README.md
+   └── __init__.py  
+   └── post_cohort_repeatative_script
+      └── __init__.py  
       └── monitoring_data_raw_schema_tables_update_script.py  
       └── raw_schema_to_intermediate_upsert_script.py  
       └── upsertion_intermediate_to_final.py  
-   └── pre_cohort_non_repeatative_script  
+   └── pre_cohort_non_repeatative_script 
+      └── __init__.py 
       └── Add_new_cohorts_names_for_upcoming_cohort.py  
       └── Update_incubator_name_based_on_email.py  
     
 ├── **old_data_insertion_scripts/**  
    └── README.md  
+   └── __init__.py
    └── data_insertion_script.py  
    └── table_creation.py  
    └── load_csvs_to_db.py  
    └── update_clean_data.py  
    └── update_course_name_INC_7_script.py  
    └── update_script_location_id.py  
```

Comment:
let's clean up the `old_data_insertion_scripts`,  `bug_fixing_on_production` folders, if you don't need them in future. Add an `archive` folder and move them if you're not sure, with a simple proper readme.

Reply:
also add `.gitignore` file and add ` __pycache__` to it, so that binary files show up here. this is applicable to ALL repositories.